Sources/Public/Animation/LottieView.swift

```diff
@@ -287,6 +304,18 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a new instance of this view, which will invoke the provided `loadAnimation` closure
+  /// whenever the `binding` value is updated.
+  ///
+  /// - Note: This function requires a valid `loadAnimation` closure provided during view initialization.
+  /// - Warning: The existing animation will be removed before calling `loadAnimation`,
+  ///            potentially rendering the provided `Placeholder`.
```

Comment:
I think a "note" is more appropriate here than a "warning"

```suggestion
  /// - Note: The existing animation will be removed before calling `loadAnimation`,
  ///         which will cause the `Placeholder` to be displayed until the new animation finishes loading.
```